src/suggestion.js

```diff
@@ -0,0 +1,42 @@
+
+export class Suggestion {
+    static create(suggestion){
+       return fetch ('https://app-js-by-yk-default-rtdb.europe-west1.firebasedatabase.app/suggestion.json', {
+            method: 'POST',
+            body: JSON.stringify(suggestion),
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        })
+        .then(response => response.json())
+        .then(response => {
+            suggestion.id = response.name
```

Comment:
This is bad mistake: name can be duplicated
Try using something else e.g. Math.random